node/hack/bridge-value-tools/main.go

```diff
@@ -0,0 +1,243 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"math"
+	"math/big"
+	"os"
+	"strconv"
+
+	"golang.org/x/text/language"
+	"golang.org/x/text/message"
+)
+
+const ACCOUNTANT_DUMP = "data/accountant.json"
+const TOKEN_ALLOW_LIST = "data/token-allowlist.json"
+const TOKEN_METADATA = "data/token_metadata.json"
+
+type (
+	// Layout of the config data for each token
+	tokenConfigEntry struct {
+		chain       uint16
+		addr        string
+		symbol      string
+		coinGeckoId string
+		decimals    int64
+		price       float64
+	}
+)
```

Comment:
```suggestion
// Layout of the config data for each token
type tokenConfigEntry struct {
	chain       uint16
	addr        string
	symbol      string
	coinGeckoId string
	decimals    int64
	price       float64
}
```

For a single variable, the extra indentation seems silly.